src/Command/Support.js

```diff
@@ -71,10 +63,10 @@ module.exports = Command.extend({
             }
 
             if (tokens[0].toLowerCase() === 'close') {
-                return await this.processDeletion(message, commandParameters);
```

Comment:
Unsure what "commandParameters" even did here, the `processDeletion` function didn't even require this.